resources/sdk/purecloudjava/tests/tests.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure if this should be added

Reply:
This shouldn't be added to the PR. You can add "resources/sdk/purecloudjava/tests/tests.iml" to the root .gitignore to have it ignored.